components/Footer.tsx

```diff
@@ -1,7 +1,10 @@
 const Footer = () => {
   return (
-    <footer className="flex justify-center items-baseline py-3">
-      <p className="text-white text-sm">Built with 💻 using NextJS, TailwindCSS and CosmosDB</p>
+    <footer className="flex justify-center items-baseline py-3 border-t-[1px] border-gray-300 mt-6">
```

Comment:
we don't need a border for footer
```suggestion
    <footer className="flex justify-center items-baseline py-3">
```